packages/fields/src/types/Text/README.md

```diff
@@ -23,3 +23,4 @@ keystone.createList('Product', {
 | ------------ | --------- | ------- | --------------------------------------------------------------- |
 | `isRequired` | `Boolean` | `false` | Does this field require a value?                                |
 | `isUnique`   | `Boolean` | `false` | Adds a unique index that allows only unique values to be stored |
+| `isMultiline` | `Boolean` | `false` | Makes the field more suitable for multiline input               |
```

Comment:
Can we be a bit more explicit about *how* it's more suitable? Perhaps explain the difference in input component in the admin UI.

Reply:
how about something like "Makes the field render as a textarea"